runner/runners/queue.go

```diff
@@ -62,13 +62,11 @@ func NewQueueRunner(
 				statusManager.UpdateService(runner.ServiceStatus{Initialized: false, Error: err})
 			} else {
 				statusManager.UpdateService(runner.ServiceStatus{Initialized: true})
+				go controller.loop()
```

Comment:
The latest merge has all requests being serviced within loop(). We need to run loop so we can reject run requests (even if they don't happen in practice due to new node init checking).